platform/ios/platform/ios/SwiftUI/MapView.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  MapView.swift
+//
+//  Copyright © 2021 MapLibre Contributors. All rights reserved.
+//
+
+import SwiftUI
+import Mapbox
+
+struct MapView: UIViewRepresentable {
+    let style: String
+    
+    func makeUIView(context: UIViewRepresentableContext<MapView>) -> MGLMapView {
+        var frame = CGRect(x: 0, y: 0, width: 10, height: 20)
+//        frame = .zero  // TODO: investigate if this case works
```

Comment:
Do we have the GitHub issue opened? We could reference it here.

Reply:
I've opened issue https://github.com/maplibre/maplibre-gl-native/issues/67.  Which will make use of this new test app.